tests/integration.rs

```diff
@@ -3135,6 +3135,63 @@ mod parallel {
         handle_child_output(r, &output);
     }
 
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    fn test_dmi_oem_strings() {
+        let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+        let guest = Guest::new(Box::new(focal));
+
+        let s1 = "io.systemd.credential:xx=yy";
+        let s2 = "This is a test string";
```

Comment:
Ah cool, I'm glad this worked, I was worried the spaces inside the `s2` could have prevented the parsing from working correctly.